vpr/src/draw/draw_basic.h

```diff
@@ -111,6 +111,10 @@ void draw_crit_path(ezgl::renderer* g);
  * 
  * This function draws critical path elements based on the provided timing paths
  * and indexes map. It is primarily used in server mode, where items are drawn upon request.
+ *
+ * @param paths The vector of TimingPath objects representing the critical paths.
```

Comment:
What are the indices of the sub-path elements to draw?  These are indices suitable to access what data structure?

Reply:
solved https://github.com/w0lek/vtr-verilog-to-routing/pull/11/commits/95948f54ea16deff9d12a33f98c92e70eb74222b
(will be updated later)